Add new info on embeddable dashboards (#26214)
DataDog/documentation: content/en/dashboards/sharing/shared_dashboards.md

By default, public dashboards are accessible for one year before they expire and switch to a **Paused** state. You can turn off or adjust the expiration date in the **Select a Share Type** step.

## Embedded dashboards

You can add embedded dashboards to a website with an iFrame. To share an embedded dashboard:
1. Click **Share** in the upper right corner of the dashboard.
1. Select **Share Dashboard**.

> 1. Select the **Public** option in the **Select a Share Type** step and acknowledge that the dashboard is accessible to anyone with the link. 

Reviewer: Sorry I made a mistake in the draft docs. 

The third bullet point here should be:
```
1. Select the **Embed** option in the **Select a Share Type** step.
```

Author: ```suggestion
1. Select the **Embed** option in the **Select a Share Type** step.
```
